### ISSUE TITLE
Can't set perf_mode_level to disabled

### ISSUE THREAD

**Issue by austin-ek** [imperva/terraform-provider-incapsula#66]
`perf_mode_level` cannot be set to `disabled` on a site resource. When I set it and run `terraform apply`, I get this error

```
Error: Error status code 406 from Incapsula service when updating Incap Performance Settings for Site ID [REDACTED]: "cannot tag response headers when caching is disabled"
```

No matter what I set `perf_response_tag_response_header` to (tried `null`, tried `""`, tried a real string), this doesn't work.

**Comment by joeymoore** [comment]
@austin-ek 
This is a bug we have and is on queue to be resolved in the next provider release.
For now, you can set the following site variable to none:
acceleration_level = "none"
This equivalent to no caching, assuming you have no cache rules.
https://docs.imperva.com/bundle/cloud-application-security/page/settings/caching-settings.htm#Setcachemode

**Comment by joeymoore** [comment]
This issue was fixed in release 2.7.4
